### PR TITLE
dataprovider relations

### PR DESCRIPTION
**Pull request by claudiocro**

### CODE REVIEW

File: packages/dataprovider/package.json [hunk 1]
```diff
@@ -25,7 +26,7 @@
     "typescript": "^3.8.3"
   },
   "dependencies": {
-    "graphql": "^15.0.0",
+    "graphql": "^14.1.1",
```
[the remark this answers is not in the document]
Comment (claudiocro) [reply packages/dataprovider/package.json:34]: relaxed to be greater than `>=14.0.0.`

**Comment by claudiocro** [comment]
@macrozone this is working for our existing projects. Codequality is not there yet, but in my opinion it would be ok to release it so we can start to use it.

Next steps would be:
* Reactivate the tests
* Refactor buildVariables
* Automatically release it with travis

File: packages/dataprovider/src/index.ts [hunk 2]
```diff
@@ -42,43 +39,37 @@ const defaultOptions = {
...
-                ...otherParams,
-              })
-            )
-          ).then((results) => {
-            return { data: results.map(({ data }: any) => data.id) };
-          });
-        }
+  return buildDataProvider(merge({}, defaultOptions, options)).then((graphQLDataProvider) => {
+    return (fetchType: string, resource: string, params: { [key: string]: any }): Promise<any> => {
+      // Temporary work-around until we make use of updateMany and deleteMany mutations
```
Comment (macrozone) [inline packages/dataprovider/src/index.ts:44]: by the way, we could acutally do that

**Pull request merged**